README.md

```diff
@@ -1,2 +1,39 @@
-# Personal-portfolio
-A list of the projects I have done over time
+# Personal-portfolio-project
+This project lists the projects I have done over time. It includes the source code for the website containing my portfolio and other supporting files. It was built for mobile platforms therefore you will get the best experience using the website. 
+
+##Built With   
+HTML and CSS
+
+## Getting Started
+This project can be acessed at my [Github account](https://github.com/Ghostman747) by anyone as it is a public repository.
+You will need a git hub account if you want to clone it and also a web browser capable of running HTML 5 for maximum quality of the content.
+The projects themselves can be accessed at the index.html file available in this repository.
+Also included is additional files for cheking the quality of the webpage and their installation directions.
+License information is also attached in this repository. Anyone is allowed to use the files accordingly as long as they use them positively.
+
+For any queries or suggestions, you can reach me at any of the channels listed below.
+
+👤 Author
+GitHub: [@Ghostman747](https://github.com/Ghostman747)
+Twitter: [@aquinasmoriasi](https://twitter.com/aquinas747)
+LinkedIn: [Aquinas Moriasi](www.linkedin.com/in/aquinas-moriasi)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ghostman747/Hello-Microverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- My coding partners and friends who helped me through the project among others.
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] This link is broken, kindly add the license file to fix the link and keep the readme professional!